src/main/java/com/crud/tasks/trello/facade/TrelloFacade.java

```diff
@@ -0,0 +1,35 @@
+package com.crud.tasks.trello.facade;
+
+import com.crud.tasks.domain.*;
+import com.crud.tasks.mapper.TrelloMapper;
+import com.crud.tasks.service.TrelloService;
+import com.crud.tasks.trello.validator.TrelloValidator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class TrelloFacade {
+    public static final Logger LOGGER = LoggerFactory.getLogger(TrelloFacade.class);
+
+    @Autowired
+    private TrelloService trelloService;
+    @Autowired
+    private TrelloMapper trelloMapper;
+    @Autowired
+    private TrelloValidator trelloValidator;
```

Comment:
spróbuj zmienić wstrzykiwanie przez pola na wstrzykiwanie przez konstruktor, https://docs.spring.io/spring-boot/docs/1.4.1.RELEASE/reference/html/using-boot-spring-beans-and-dependency-injection.html